files/fr/web/css/@font-face/ascent-override/index.html

```diff
@@ -0,0 +1,74 @@
+---
+title: ascent-override
+slug: Web/CSS/@font-face/ascent-override
+browser-compat: css.at-rules.font-face.ascent-override
+translation_of: 'Web/CSS/@font-face/ascent-override'
+---
+<p>{{CSSRef}}</p>
+
+<p>Le descripteur CSS <strong><code>ascent-override</code></strong> définit la mesure du jambage supérieur (ascendant) de la police. La mesure du jambage supérieur correspond à la distance verticale par rapport à la hauteur d'X (hauteur du caractère bas de casse, à l'exclusion des jambages inférieurs et supérieurs) que CSS doit utiliser pour agencer les boîtes dans un contexte de formatage en ligne.</p>
+
+<h2 id="syntax">Syntaxe</h2>
+
+<pre class="brush: css">ascent-override: normal;
+ascent-override: 90%;</pre>
+
+<h3 id="values">Valeurs</h3>
+
+<dl>
+  <dt><code>normal</code></dt>
+  <dd>La valeur par défaut. La mesure du jambage supérieur est obtenue à partir du fichier de police.</dd>
+  <dt><code>&lt;percentage&gt;</code></dt>
+  <dd>Une valeur exprimée en {{cssxref("&lt;percentage&gt;", "&lt;pourcentage&gt;")}}.</dd>
```

Comment:
As discussed on Matrix, I think it's better to leave cssxref calls for CSS types in English.
However, that should not prevent us from writing stuff like "Une valeur exprimée en pourcentage (type {{cssxref("&amp;lt;percentage&amp;gt;)}} qui traduit l'augmentation ou la réduction par rapport à la taille de la police ({{cssxref("font-size")}}."

I find the English version a bit dry here as there is no baseline for what the percentage is relative too so I'd include a few additional explanations. I'll leave it up to you if you want to fix upstream  ^- ^

Reply:
Fixed in https://github.com/mdn/translated-content/pull/1752/commits/600211688c290bd6c766b2a1f9b7bf8f505cd3cf and https://github.com/mdn/translated-content/pull/1752/commits/3be2f814cbedad8a12671dce7e74be7533b451cd